ci/docker/aarch64-musl/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ghcr.io/cross-rs/aarch64-unknown-linux-musl
+
+RUN apt-get update -y && apt-get install -y ninja-build
+RUN git config --global --add safe.directory '*'
+ENV EXTRA_RUSTFLAGS=-Ctarget-feature=-crt-static
```

Comment:
This is a bit different from Wasmtime in that `EXTRA_RUSTFLAGS` isn't recognized in this repository, could this be updated to `RUSTFLAGS` and/or could `EXTRA_RUSTFLAGS` get handled? (as-is static binaries are produced when it's intended that dynamic binaries are produced instead)